fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexScrubberTest.java

```diff
@@ -72,6 +75,7 @@ void testScrubberSimpleMissing() throws ExecutionException, InterruptedException
                 .setScrubbingPolicy(OnlineIndexScrubber.ScrubbingPolicy.newBuilder()
                         .setLogWarningsLimit(Integer.MAX_VALUE)
                         .setAllowRepair(false)
+                        .ignoreIndexTypeCheck(legacy)
```

Comment:
If `legacy=true`, the scrubber will go through the old code.

Reply:
That does seem right to make those two controls the same.
As you have it, if you call `ignoreIndexTypeCheck(true)` it will run with the old code, on any index type. If you call `ignoreIndexTypeCheck(false)` (or don't call it at all), it will run with the new code.
That seems like a bad sed of options, and that we still want the option to run the old code, and fail if it's not a value index. 

I think I got it in my head that it was the other way around, and that there would no longer be a way to run the old code against non-value indexes. I don't know why that option was added, presumably a good reason, perhaps to run against an index type that is close enough to value index that it is ok, but without actually adding support for that index type. Now that we have this new framework, we should just add support. (to be clear, I'm not advocating for adding support for any other index types to this PR).

Perhaps a new option on the policy is appropriate.

Reply:
✅